arch/riscv/core/coredump.c

```diff
@@ -116,3 +117,21 @@ uint16_t arch_coredump_tgt_code_get(void)
 {
 	return COREDUMP_TGT_RISC_V;
 }
+
+#if defined(CONFIG_DEBUG_COREDUMP_DUMP_THREAD_PRIV_STACK)
+void arch_coredump_priv_stack_dump(struct k_thread *thread)
```

Comment:
There's a similar implementation in `stacktrace.c`

https://github.com/zephyrproject-rtos/zephyr/blob/74c5c84c66d59320ab39b58662d60e6cb6f188c2/arch/riscv/core/stacktrace.c#L55-L71

Reply:
Thanks. Updated.